shared/reducers/__tests__/fs.js

```diff
@@ -148,13 +150,64 @@ describe('fs reducer', () => {
             Constants.makeFolder({
               children: I.Set(['file1']),
               name: 'folder0',
+              prefetchStatus: Constants.prefetchNotStarted,
               progress: 'loaded',
             }),
           ],
         ]),
       })
     )
-    expect(stage1.pathItems).not.toBe(state0.pathItems)
-    expect(getFolderOrFail(stage1.pathItems, folder0Path).children).toEqual(I.Set(['file1']))
+    expect(state1.pathItems).not.toBe(state0.pathItems)
+    expect(getFolderOrFail(state1.pathItems, folder0Path).children).toEqual(I.Set(['file1']))
+    expect(getFolderOrFail(state1.pathItems, folder0Path).prefetchStatus).toBe(Constants.prefetchNotStarted)
```

Comment:
you can compare `I.Record` like this?

Reply:
`toBe` is reference equality. So this is saying they should be exactly the same object.